.github/workflows/build-on-prem.yml

```diff
@@ -25,4 +25,4 @@ jobs:
             ${{ runner.os }}-maven-
 
       - name: Build for AEM 6.5
-        run: mvn --batch-mode --update-snapshots clean install
```

Comment:
rename to build and update README with new badge and delete deleted workflow badge